src/rp2_common/pico_cyw43_driver/cyw43_firmware.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024 Raspberry Pi (Trading) Ltd.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import sys
+
+with open(sys.argv[1], "r") as f:
+    data = f.read()
+    lines = data.split(";")
+    for line in lines[1].split("\n"):
```

Comment:
I guess you could `break` from this `for` loop once both `cyw43_wifi_fw_len` and cyw43_clm_len` have been read?